proto/agglayer/prover/v1/proof_generation.proto

```diff
@@ -43,3 +43,33 @@ message GenerateProofError {
   // The kind of error that occurred.
   ErrorKind error_type = 2;
 }
+
+// Service for fetching auth proof.
+service AuthProofService {
```

Comment:
This service shouldn't be in this file.
This file represents the API definition of the `agglayer-prover`.

It should be placed into `proto/aggkit/prover/v1/auth_proof_generation.proto` I guess